api/templates/design.go

```diff
@@ -242,6 +242,7 @@ var _ = Service("templates", func() {
 			templateAttr()
 			versionAttr()
 			branchAttr()
+			instanceNameAttr()
```

Comment:
Pridane `name` do payload v `/use` endpointe, malo to byt az v dalsej verzii, no nakoniec je to vo `v1`.